helix-core/src/test/java/org/apache/helix/task/TestGetLastScheduledTaskExecInfo.java

```diff
@@ -41,37 +42,47 @@ public void beforeClass() throws Exception {
   }
 
   @Test
-  public void testGetLastScheduledTaskExecInfo() throws InterruptedException {
-    List<Long> startTimesWithStuckTasks = setupTasks("TestWorkflow_2", 5, 99999999);
+  public void testGetLastScheduledTaskExecInfo() throws Exception {
+    // Start new queue that has one job with long tasks and record start time of the tasks
+    String queueName = TestHelper.getTestMethodName();
+    List<Long> startTimesWithStuckTasks = setupTasks(queueName, 5, 99999999, 2);
```

Comment:
Nit: could we make 99999999 a constant and name it something more appropriate?

Reply:
Addressed.